easybuild/easyconfigs/x/Xmipp/Xmipp-3.19.02-fosscuda-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,95 @@
+name = 'Xmipp'
+version = '3.19.02'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/I2PC/scipion/wiki'
+description = """Scipion is an image processing framework to obtain 3D
+models of macromolecular complexes using Electron Microscopy (3DEM). It
+integrates several software packages and presents an unified interface
+for both biologists and developers. Scipion allows to execute workflows
+combining different software tools, while taking care of formats and
+conversions. Additionally, all steps are tracked and can be reproduced
+later on. 
+"""
+
+toolchain = {'name': 'fosscuda', 'version': '2019a'}
+# toolchainopts = {'openmp': True, 'noopt': True, 'debug': True }
+
+sources = [
+    {
+        'source_urls': ['https://github.com/I2PC/xmipp/archive/'],
+        'download_filename': 'v%(version)s.tar.gz',
+        'filename': '%(namelower)s-v%(version)s.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/I2PC/xmippCore/archive/'],
+        'download_filename': 'v%(version)s.tar.gz',
+        'filename': 'xmippCore-v%(version)s.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/I2PC/xmippViz/archive/'],
+        'download_filename': 'v%(version)s.tar.gz',
+        'filename': 'xmippViz-v%(version)s.tar.gz',
+    },
+    {
+        'git_config': {
```

Comment:
using `git_config` causes trouble for checksums (see https://github.com/easybuilders/easybuild-framework/issues/2726)

But you don't need this at all when downloading from GitHub, see https://github.com/DStrelak/cuFFTAdvisor/archive/14523b8ca7e2f5474feae49eb5ac8ca1fa25f5cd.tar.gz.

So:

```python
{
    'source_urls': ['https://github.com/DStrelak/cuFFTAdvisor/archive/'],
    'download_filename': '14523b8ca7e2f5474feae49eb5ac8ca1fa25f5cd.tar.gz',
    'filename': 'cuFFTAdvisor-20191010.tar.gz',
}

Reply:
oops, didn't think of that...